test/index.spec.mjs

```diff
@@ -197,5 +197,11 @@ for (const base of ['index.js', 'index.mjs']) {
         assert(warframe?.components?.length > 0);
       });
     });
+
+    it('marketCost should be a number when present', () => {
+      data.items.filter(i => !namedExclusions.includes(i.marketCost)).forEach((item) => {
+        assert(item.marketCost)
+      })
+    }) 
```

Comment:
I would probably do something more like this, it doesn't make sense to check for excalibur prime in this case also we want a more clear error message for when something goes wrong

```suggestion
    data.items.forEach((item) => {
      it(`${item.name} marketCost should be a number if present`, () => {
        assert(['number', 'undefined'].includes(typeof item.marketCost));
      });
```
or
```suggestion
    data.items
      .filter((item) => typeof item !== 'undefined')
      .forEach((item) => {
        it(`${item.name} marketCost should be a number if present`, () => {
          assert(typeof item.marketCost === 'number');
        });
      });
```

Reply:
i would move this into the existing foreach and just unwrap it

Reply:
i don't think these tests are working at all anyway, so i'm gonna work on this for you, @SlayerOrnstein, and we'll pair on it later

Reply:
Oh okay so I wasn't going crazy because the test above that should have outputted the names for the items being tested and I didn't think anything was limiting the logger.

Reply:
> I would probably do something more like this, it doesn't make sense to check for excalibur prime in this case also we want a more clear error message for when something goes wrong

So the reason for this was more to just write something so @TobiTenno can take a look, was never actually supposed to run, sorry about that